pkg/domain/entities/container_ps.go

```diff
@@ -36,6 +36,8 @@ type ListContainer struct {
 	ImageID string
 	// If this container is a Pod infra container
 	IsInfra bool
+	// Label specified for container
+	Label string
```

Comment:
This is not used.

Reply:
Yup this was an attempt to get around the man page problem.  Removed.